<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>531</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>QuizzCard : L'application de révision de cartes</string>
    </property>
   </widget>
   <widget class="QListWidget" name="liste_Paquets">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>281</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Liste de mes paquets</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reviser">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Réviser un paquet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="supprPaquet">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer un paquet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ajoutPaquet">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter un paquet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ajoutCarte">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>180</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter une carte</string>
    </property>
   </widget>
   <widget class="QPushButton" name="supprCarte">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>270</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer une carte</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connexion">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="import_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>320</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Importer Paquet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="export_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>350</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Exporter Paquet</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bienvenue :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>598</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionConnexion"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionImporter"/>
    <addaction name="actionExporter"/>
   </widget>
   <widget class="QMenu" name="menuAjouts">
    <property name="title">
     <string>Création</string>
    </property>
    <addaction name="actionAjouter_un_paquet"/>
    <addaction name="actionAjouter_une_carte"/>
   </widget>
   <widget class="QMenu" name="menuSuppression">
    <property name="title">
     <string>Suppression</string>
    </property>
    <addaction name="actionSupprimession_paquet"/>
    <addaction name="actionSuppression_carte"/>
   </widget>
   <widget class="QMenu" name="menuAffichage">
    <property name="title">
     <string>Affichage</string>
    </property>
    <addaction name="actionAffichage_statistiques"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
   <addaction name="menuAffichage"/>
   <addaction name="menuAjouts"/>
   <addaction name="menuSuppression"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionImporter">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Importer</string>
   </property>
  </action>
  <action name="actionExporter">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Exporter</string>
   </property>
  </action>
  <action name="actionAjouter_un_paquet">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Ajouter un paquet</string>
   </property>
  </action>
  <action name="actionAjouter_une_carte">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Ajouter une carte</string>
   </property>
  </action>
  <action name="actionSupprimession_paquet">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Supprimession paquet</string>
   </property>
  </action>
  <action name="actionSuppression_carte">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Suppression carte</string>
   </property>
  </action>
  <action name="actionConnexion">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Se connecter</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionAffichage_statistiques">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Affichage statistiques</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
